src/rust/grapl-web-ui/src/main.rs

```diff
@@ -31,7 +31,7 @@ async fn main() -> Result<(), GraplUiError> {
     HttpServer::new(move || {
         App::new()
             .wrap(actix_web::middleware::Logger::default())
-            .wrap(actix_web::middleware::Compress::default())
+            // .wrap(actix_web::middleware::Compress::default())  // todo: Reenable compression when brotli isn't vulnerable
```

Comment:
For long term todos like this pls add a github issue link:
https://github.com/grapl-security/issue-tracker/issues/803